metrics/scaling/net-serve.json.in

```diff
@@ -0,0 +1,39 @@
+{
+    "apiVersion": "apps/v1",
+    "kind": "Deployment",
+    "metadata": {
+        "labels": {
+            "run": "net-serve"
+        },
+        "name": "@DEPLOYMENT@"
+    },
+    "spec": {
+        "replicas": @REPLICAS@,
+        "selector": {
+            "matchLabels": {
+                "run": "net-serve"
+            }
+        },
+        "template": {
+            "metadata": {
+                "labels": {
+                    "run": "net-serve",
+                    "@LABEL@": "@LABELVALUE@"
+                }
+            },
+            "spec": {
+                "terminationGracePeriodSeconds": @GRACE@,
+                "runtimeClassName": "@RUNTIMECLASS@",
+                "automountServiceAccountToken": false,
+                "containers": [{
+                    "name": "net-serve",
+                    "image": "dklyle/net-serve",
```

Comment:
we should find a generic way to either host this pod or find an existing one we can use (there are some k8s e2e test pods I think may be suitable, for instance the kata k8s pod http liveness test uses the minimal k8s test pod I believe: https://github.com/kata-containers/tests/blob/master/integration/kubernetes/runtimeclass_workloads/pod-http-liveness.yaml

Reply:
Agree. 
I'll check tomorrow the deployment that you're pointing out and check if we can used it instead of `dklyle/net-serve` and update this WIP branch accordingly, oki doki?

Reply:
any luck swapping to the non @dklyle image @marcemq ?